core/src/main/java/bisq/core/btc/TxFeeEstimationService.java

```diff
@@ -87,14 +102,14 @@ public TxFeeEstimationService(FeeService feeService,
                                                            BtcWalletService btcWalletService,
                                                            Preferences preferences) {
         Coin txFeePerByte = feeService.getTxFeePerByte();
-        // We start with min taker fee size of 260
+        // We start with min taker fee size of 175
         int estimatedTxSize = TYPICAL_TX_WITH_1_INPUT_SIZE;
         try {
             estimatedTxSize = getEstimatedTxSize(List.of(tradeFee, amount), estimatedTxSize, txFeePerByte, btcWalletService);
         } catch (InsufficientMoneyException e) {
             if (isTaker) {
-                // if we cannot do the estimation we use the payout tx size
-                estimatedTxSize = PAYOUT_TX_SIZE;
+                // if we cannot do the estimation we use the deposit tx size
```

Comment:
Maybe change comment to: "if we cannot do the estimation we use the largest of our txs which is the deposit tx

Reply:
Done